django-mixin/dashboards/django-overview.json

```
@@ -0,0 +1,1334 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Django metrics dashboard using korfuri/django-prometheus Prometheus metrics exporter",
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "gnetId": 9528,
+  "graphTooltip": 0,
+  "id": 2,
+  "iteration": 1642775384375,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "collapsed": false,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 32,
+      "panels": [],
+      "title": "Database",
+      "type": "row"
+    },
+    {
+      "description": "Database connections created per second, per vendor.",
```

As commented below. It would be easier to specify the data source on the panel, rather than on each target individually.

This is true for most (all?) panels in the dashboard, but only commenting on this one.

```suggestion
      "description": "Database connections created per second, per vendor.",
      "datasource": "$datasource",
```

---

Moved datasource to all panels instead. Thanks!